nox.py

```diff
@@ -30,13 +30,13 @@ def docs(session):
     # Install Sphinx and also all of the google-cloud-* packages.
     session.chdir(os.path.realpath(os.path.dirname(__file__)))
     session.install('Sphinx >= 1.6.2', 'sphinx_rtd_theme')
+    session.install('.')
```

Comment:
Can we just kill off installing the umbrella package altogether?  Its only effect is to install the subpackages from PyPI / wheel cache, which we then immediately overwrite with the local versions.